projects/media-viewer/src/lib/annotations/tags/tags.component.ts

```diff
@@ -7,7 +7,7 @@ import {Observable} from 'rxjs';
 @Component({
   selector: 'mv-tags',
   templateUrl: './tags.component.html',
-  styleUrls: ['./tags.component.scss']
+  encapsulation: ViewEncapsulation.None
```

Comment:
is this needed if there are no styles defined in the component?

Reply:
yes of course... as it might not inherit the css